cola/themes.py

```diff
@@ -553,6 +571,21 @@ def get_all_themes():
         ),
     ]
 
+    # check if themes path exists in user folder
+    path = resources.config_home('themes')
+    if not os.path.exists(path):
+        return themes
+
+    # get only files with extension .qss
+    for root, dirs, files in os.walk(path):
+        for file in files:
+            name, ext = file.split('.')
```

Comment:
```
name, ext = os.path.splitext(file)
```
is more appropriate because it handles filenames like `foo.bar.qss`. This code will blow up if it sees that filename. Another alternative would be `name, ext = file.rsplit('.', 1)` but splitext is a little clearer

splitext is more robust because it also won't trip up over files that don't contain `.` and will always return 2 elements. That edge case is demonstrated by `name, ext = 'file-with-no-extension'.split('.')` because the resulting tuple has only 1 element but the left-hand `name, ext =` wants 2.

Also (this one is annoying) I'd suggest avoiding the name `file` for variables. That one is so annoying because I totally want to use the name `file` all the time. The suggested alternative is `filename`.

That's actually a python2-ism.. `file` is a builtin symbol in python2. In python3 it's not, so I guess it's okay to start using that name now. Well.. okay that's a good python3 change I guess.

Anyways.. since we need to tweak this for the `splitext` thing then we might as well rename the variable to `filename` while we're here so that it doesn't get caught by any of the linters.

Do note that `splitext` will return `.qss` so we'd have to adjust the conditional below in tandem.

Reply:
done, it is better indeed